packages/frontend/index.d.ts

```diff
@@ -220,7 +220,7 @@ interface ConfigType {
     sentryPublicApiKey: string;
     sentryHost: string;
     switches: { [key: string]: boolean };
-    abTests?: { [key: string]: string };
+    abTests: { [key: string]: string };
```

Comment:
If we do this, then we should run `make gen-schema` to re-generate the schema and then commit that. Unfortunately this is not automatic at the moment.